CGPs/cgp-0040.md

```diff
@@ -0,0 +1,52 @@
+status: DRAFT
+date-executed: [if executed] <date created on, in ISO 8601 (yyyy-mm-dd) format>
+governance-proposal-id: [if submitted]
+---
+## Overview
+
+This governance proposal suggests a reduction of the fee charged for cEUR and cUSD exchanges via Mento, the Celo stability protocol.
+
+Since the activation of cUSD and cEUR via governance proposals [CGP007](https://github.com/celo-org/governance/blob/main/CGPs/cgp-0007.md) and [CGP24](https://github.com/celo-org/governance/blob/main/CGPs/cgp-0024.md), the Celo Reserve has charged a fee of 0.5%, a value controlled via the `spread` parameter in the Exchange contract, on all trades with the Celo Reserve. This fee is supposed to compensate the Celo Reserve for taking on price risk as a liquidity provider.
+
+### Status
+- DRAFT = Feedback collection
+
+A 0.5% spread effectively leads to an arbitrage free range on the cUSD/USD and cEUR/EUR rate of at least 0.5% in both directions. Additional fees an arbitrageur encounters, like fees at centralized exchanges, gas costs etc., further increase the range in which cUSD/USD and cEUR/EUR can fluctuate before arbitrage incentives are present that create pressure towards an exchange rate of 1:1.
+
+With the sharp increase in cUSD and cEUR demand seen over the last few weeks, the price of cUSD and cEUR has, as expected, been trading mainly at the upper bound of this arbitrage free zone, staying below 1.01 most of the time. To tighten the peg further, we suggest reducing the spread to 0.25% which can be expected to reduce the arbitrage free range by about 0.25% in each direction. The downside of a spread reduction is a lower fee for the reserve for any given trade as well as a larger loss for the reserve in case of an imprecise oracle rate. We however believe that the expected tighter peg of cUSD and cEUR and expected larger total trading amounts more than outweigh these potential downsides.
+
+## Proposed Changes
+
+1. Set cUSD Spread to 0.25%
+  - Destination: Exchange (Proxy: 0x67316300f17f063085Ca8bCa4bd3f7a5a3C66275; Implementation: 0xEDF3F7e01037e4583de2659C5e243621Ea2501A4), [setSpread](https://github.com/celo-org/celo-monorepo/blob/master/packages/protocol/contracts/stability/Exchange.sol#L287)
+  - Data: newSpread = 2500000000000000000000 (0.0025 * 10^24 = 2.5e21) 
+  - Value: 0 (NA)
+
+2. Set cEUR Spread to 0.25%
+  - Destination: ExchangeEUR (Proxy: 0xE383394B913d7302c49F794C7d3243c429d53D1d; Implementation: 0x622833AB6E9501C9072d2c706c60AaB5Ff0234d9), [setSpread](https://github.com/celo-org/celo-monorepo/blob/master/packages/protocol/contracts/stability/Exchange.sol#L287)
+  - Data: newSpread = 2500000000000000000000 (0.0025 * 10^24 = 2.5e21) 
+  - Value: 0 (NA)
+
+## Verification
+
+1. Confirm proposal steps: run `celocli governance:view --proposalID X` // TODO add proposal ID once submitted proposal
+2. Confirm Exchange addresses: run `celocli network:contracts`
```

Comment:
this step is also not entirely necessary
the decoding in `celocli governance:view` will only succeed if the registry contains the target addresses